Sources/SpeziViews/ViewModifier/SkeletonLoading.swift

```diff
@@ -0,0 +1,67 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+public struct SkeletonLoading<SkeletonCell: View>: View {
```

Comment:
Maybe we can get a least maybe a short "Overview" description such that something shows up in the DocC section. 
Maybe it is worth moving the detailed example to the here just because it might be more discoverable for people?